### PR TITLE
Fixes new cops not spawning with messenger bags

### LINKED ISSUE #14311: Ops don't spawn with a backpack if you have the messenger bag selected

### PR DESCRIPTION
**Pull request by jknpj**
Closes #14311.

### CODE REVIEW

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -281,11 +281,13 @@
...
-	if(synd_mob.backbag == 3 || synd_mob.backbag == 4)
-		synd_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel_sec(synd_mob), slot_back)
-	//if(synd_mob.backbag == 4) synd_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel(synd_mob), slot_back)
+	switch(synd_mob.backbag)
+		if(2)
+			synd_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/security(synd_mob), slot_back)
+		if(3,4)
+			synd_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel_sec(synd_mob), slot_back)
+		if(5)
+			synd_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/messenger/sec(synd_mob), slot_back)
```
Comment (N3X15) [inline code/game/gamemodes/nuclear/nuclear.dm:290]: magic numbers suck

Comment (jknpj) [reply code/game/gamemodes/nuclear/nuclear.dm:290]: BUT DAD

Comment (N3X15) [reply code/game/gamemodes/nuclear/nuclear.dm:290]: I know, not your fault.

They just suck.

Comment (jknpj) [reply code/game/gamemodes/nuclear/nuclear.dm:290]: Might as well do it.

**Comment by SonixApache** [comment]
i hate titles that just say 'fixes #x'

**Comment by Kurfursten** [comment]
It could be worse, his commit message makes it clear what this one does

CERTAIN CONTRIBUTORS are known for really unhelpful commit messages. And by certain contributors, I mean 90% of the repo's contributors.

**Pull request merged**